tests/filters/test_addslashes.py

```diff
@@ -0,0 +1,63 @@
+"""
+Test cases adapted from
+https://github.com/django/django/blob/main/tests/template_tests/filter_tests/test_addslashes.py
+"""
+
+import pytest
+from django.template import engines
+from django.utils.safestring import mark_safe
+
+
+@pytest.mark.xfail(reason="autoescape not there yet")
+def test_addslashes01(self):
+    """
+    @setup(
+        {
+            "addslashes01": (
+                "{% autoescape off %}{{ a|addslashes }} {{ b|addslashes }}"
+                "{% endautoescape %}"
+            )
+        }
+    )
+    """
+    output = self.engine.render_to_string(
+        "addslashes01", {"a": "<a>'", "b": mark_safe("<a>'")}
+    )
+    self.assertEqual(output, r"<a>\' <a>\'")
+
+
+@pytest.mark.xfail(reason="autoescape not there yet")
```

Comment:
I added these as xfails for the time being, marking them as accepted failures, and thus a flag to go back to these tests later on.